server/index.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const app = express();
+
+const fetch = require('node-fetch');
+const queryString = require('query-string');
+
+// for every URL path that starts with /api/, send request to upstream API service
+function peachworksAPIURL(query_obj) {
```

Comment:
hmm? this function / comment sound like it's meant to be generic for all api requests, but then it returns specifically for wtm_recipes?